app/views/users/nbp_wallet/connect.html.slim

```diff
@@ -5,15 +5,21 @@
 
 .row
   .col-8.col-md-4.col-lg-3
-    = link_to @template.url
-      = image_tag @template.qr_code_path, data: {'template-validity': @template.remaining_validity&.seconds}, alt: t('.qr_code_alt_text'), class: 'img-fluid nbp_wallet_qr_code'
-    = button_tag class: 'btn btn-primary regenerate-qr-code-button w-100 mt-3' do
+    = link_to @relationship_template.url
+      = image_tag @relationship_template.qr_code_path,
+        data: {id: 'nbp_wallet_qr_code', 'remaining-validity': @relationship_template.remaining_validity},
+        alt: t('.qr_code_alt_text'),
+        class: 'img-responsive'
+    .buttons.btn.btn-primary.w-100.mt-3 data-behavior='reload-on-click'
```

Comment:
I cannot find a definition for the class `buttons` (neither within our code, nor Bootstrap). What should it do?

On that same note (and for my personal learning): Why is a div beneficial over a real `button`? Because it doesn't "submit" the (non-existing) form?